### DISCUSSION TITLE
Just a standard language independent of the browser language

### DISCUSSION THREAD

**Discussion by brendofreitas** [mosparo/mosparo#141] [Ideas]
Is there any way I can choose a default language independent of the browser language? By default, I want to choose English and if the browser uses German, I want it to appear in English.

If you don't have this function, it would be very interesting to have.

**Comment by zepich** [comment]
Hi @brendofreitas 

Thank you very much for your idea.

Do you mean you want to define which languages are 'allowed' to be used in the frontend box?

We don't have such a feature right now, but it's a good idea and extension to be able to set the allowed languages!

In general, our primary goal is maximum accessibility, so if someone cannot understand English well, it makes more sense to show the language the user understands better (the browser language). But it looks weird if you have a website available in one language, and then the mosparo box appears in a different language.

Kind regards,

zepich

**Comment by brendofreitas** [comment]
It would be interesting if I don't have a supported language if it presents a language in English or any other language in which I can define it.

Today in testing I realized that if I don't have this language supported in the language definitions it gives an error.

**Comment by zepich** [comment]
Okay, can you tell me the exact error you had? 

In general, it should fall back to English if your browser has a language for which mosparo does not have a translation.

**Comment by zepich** [comment]
This feature was added with v1.1.0.
